sqlx-core/src/postgres/types/ltree.rs

```diff
@@ -0,0 +1,80 @@
+use crate::decode::Decode;
+use crate::encode::{Encode, IsNull};
+use crate::error::{BoxDynError, Error};
+use crate::postgres::{PgArgumentBuffer, PgTypeInfo, PgValueFormat, PgValueRef, Postgres};
+use crate::types::Type;
+use std::fmt::{self, Display, Formatter};
+use std::str::FromStr;
+
+#[derive(Clone, Debug, Default, PartialEq)]
+pub struct PgLTree {
+    labels: Vec<String>,
```

Comment:
Just a question. Isn't ltree a simple string?

Reply:
Basically yes. It might be a bit different to query for but in the end it's a lost of labels joined by dots